backend/src/main/java/com/soen341/instagram/service/impl/PictureService.java

```diff
@@ -101,13 +115,16 @@ private Picture getPictureFromId(String id) {
         } catch (NumberFormatException e) {
             throw new InvalidIdException("Invalid picture ID.");
         }
+        return getPictureFromId(pictureId);
+    }
+    private Picture getPictureFromId(long pictureId) {
         Optional<Picture> optionalPic = pictureRepository.findById(pictureId);
         if (!optionalPic.isPresent()) {
             throw new PictureNotFoundException("The specified picture does not exist.");
         }
         return optionalPic.get();
     }
-
+    
```

Comment:
Remove

Reply:
i just checked this and there is no space, or do u want me to glue all the methods together in a bloc?
![image](https://user-images.githubusercontent.com/45129406/76908957-5fc3ee00-6880-11ea-9e7a-81da6715556c.png)